packages/playwright-msw/src/utils.test.ts

```diff
@@ -81,58 +81,53 @@ describe('utils', () => {
   });
 
   describe('convertMswPathToPlaywrightUrl', () => {
-    it.each([
-      [
-        'should return a regex without modifying it',
-        {
-          path: /^\/api\/.*/,
-          expected: /^\/api\/.*/,
-        },
-      ],
-      [
-        'should replace a single route parameter with a wildcard if provided',
-        {
-          path: '/users/:userId',
-          expected: '/users/*',
-        },
-      ],
-      [
-        'should return path with wildcard on the end even if there are no route parameters',
-        {
-          path: '/users/123',
-          expected: '/users/123*',
-        },
-      ],
-      [
-        'should replace multiple sequential route parameters if provided',
-        {
-          path: '/users/:userId/:something',
-          expected: '/users/*/*',
-        },
-      ],
-      [
-        'should replace multiple disjointed route parameters if provided',
-        {
-          path: '/users/:userId/photos/:photoId',
-          expected: '/users/*/photos/*',
-        },
-      ],
-      [
-        'should drop query parameters',
-        {
-          path: '/users/:userId/photos/:photoId?potato=123',
-          expected: '/users/*/photos/*',
-        },
-      ],
-      [
-        'should add a wildcard on the end so that query parameters can be matched',
-        {
-          path: '/api/v1/documents/?potato=123',
-          expected: '/api/v1/documents/*',
-        },
-      ],
-    ])('%s', (_, { path, expected }) => {
-      expect(convertMswPathToPlaywrightUrl(path)).toStrictEqual(expected);
+    it.each<{ path: string; url: string; expected: boolean }>`
```

Comment:
@lesha1201 thanks for helping point out the glob matching issue. I've written a number of tests to help ensure that this `convertMswPathToPlaywrightUrl` matches the expected paths. Do all of these tests align with your expectations?

Reply:
@valendres I probably won't be able to review it today so I'll try to review it on Monday. If you would like to merge then please don't wait me. I can review it after and we can create a new PR of needed. Thanks for quick fixes.